sys-devel/llvm-common/llvm-common-14.0.6.ebuild

```
@@ -1,28 +1,58 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-inherit llvm.org
+inherit elisp-common llvm.org
 
 DESCRIPTION="Common files shared between multiple slots of LLVM"
 HOMEPAGE="https://llvm.org/"
 
 LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA"
 SLOT="0"
 KEYWORDS="amd64 arm arm64 ~ppc ppc64 ~riscv sparc x86 ~amd64-linux ~ppc-macos ~x64-macos"
+IUSE="emacs"
 
 RDEPEND="
 	!sys-devel/llvm:0
 "
+BDEPEND="
+	emacs? ( app-editors/emacs )
```

```suggestion
	emacs? ( >=app-editors/emacs-23.1:* )
```
Please use the dependency as suggested in elisp-common documentation.